index.js

```diff
@@ -177,8 +177,8 @@
 
     _valueHandler(value) {
       try {
-        const jsonValue = JSON.parse(value);
-        const valueToSend = value !== EMPTY ? (jsonValue.value || jsonValue) : null;
+        const jsonValue = value !== EMPTY ? JSON.parse(value) : null;
+        const valueToSend = jsonValue ? jsonValue.value : null;
```

Comment:
I think we might wanna throw here if `value !== EMPTY` and `jsonValue` is falsy, but this is fine too.